common-pg/src/main/scala/com/socrata/pg/store/PGSecondaryDatasetMapReader.scala

```diff
@@ -1,53 +1,167 @@
 package com.socrata.pg.store
 
-import java.sql.Connection
 import com.rojoma.simplearm.v2._
-import com.socrata.datacoordinator.id.{DatasetId, RollupName}
+import com.socrata.datacoordinator.id.{CopyId, DatasetId, RollupName}
+import com.socrata.datacoordinator.truth.metadata.sql.{BasePostgresDatasetMapReader, PostgresDatasetMapReader}
+import com.socrata.datacoordinator.truth.metadata.{CopyInfo, DatasetInfo, LifecycleStage, TypeNamespace}
 import com.socrata.datacoordinator.util.TimingReport
-import com.socrata.datacoordinator.truth.metadata.sql.{PostgresDatasetMapReader, BasePostgresDatasetMapReader}
-import com.socrata.datacoordinator.truth.metadata.{TypeNamespace, CopyInfo}
-import com.socrata.soql.types.SoQLType
+import org.joda.time.DateTime
+
+import java.sql.Connection
+import scala.reflect.ClassTag
 
-trait LocalRollupReaderOverride[CT] { self: BasePostgresDatasetMapReader[CT] =>
-  def localRollupsQuery = "SELECT name, soql, table_name FROM rollup_map WHERE copy_system_id = ?"
+trait LocalRollupReaderOverride[CT] {
+  self: BasePostgresDatasetMapReader[CT] =>
   override def rollups(copyInfo: CopyInfo): Seq[LocalRollupInfo] = {
     using(conn.prepareStatement(localRollupsQuery)) { stmt =>
```

Comment:
can we move the localRollupsQuery back up here since this is where it's being executed?